providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/process/model/FeatureFlag.java

```diff
@@ -0,0 +1,40 @@
+package dev.openfeature.contrib.providers.flagd.resolver.process.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import lombok.Getter;
+
+import java.util.Map;
+
+/**
+ * flagd feature flag model.
+ */
+@Getter
+public class FeatureFlag {
+    private final String state;
+    private final String defaultVariant;
+    private final Map<String, Object> variants;
+    private final String targeting;
+
+    /**
+     * Construct a flagd feature flag.
+     */
+    @JsonCreator
+    public FeatureFlag(@JsonProperty("state") String state,
+                       @JsonProperty("defaultVariant") String defaultVariant,
+                       @JsonProperty("variants") Map<String, Object> variants,
+                       @JsonProperty("targeting") @JsonDeserialize(using = StringSerializer.class) String targeting) {
+        this.state = state;
+        this.defaultVariant = defaultVariant;
+        this.variants = variants;
+        this.targeting = targeting;
+    }
+
+    /**
+     * Get targeting rule of the flag.
+     */
+    public String getTargeting() {
+        return this.targeting == null ? "{}" : this.targeting;
```

Comment:
maybe let's extract `{}` into a public static final class field so it can be used also in [InProcessResolver.java](https://github.com/open-feature/java-sdk-contrib/pull/412/files#diff-27f34c6acd23d899adf4c27d00f4d5b9e730090bb2a88efc9a286f3da0bd9dcc)

Reply:
this is done :)